### ISSUE TITLE
Missing documentation for "shield resistance"

### ISSUE THREAD

**Issue by tibetiroka** [endless-sky/endless-sky#9105]
### Location

wiki

### Describe the issue

The wiki doesn't list the `shield resistance` attribute, but it is used in `drak/indigenous.txt`.

### Expected content

It should be on [this page](https://github.com/endless-sky/endless-sky/wiki/CreatingOutfits).

Also, is there `hull resistance`, or `shield|hull resistance energy|heat|fuel`?

**Comment by lumbar527** [comment]
What does `shield resistance` do?

**Comment by tibetiroka** [comment]
No idea.

**Comment by mOctave** [comment]
I think that what it does is add a flat deduction to any damage given to the ship's shields, although someone more knowledgeable should affirm or deny this.

**Comment by Quantumshark** [comment]
I don't think it does anything. `shield resistance` doesn't appear anywhere in the code, so it won't have any effect.
